src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorHost.cs

```diff
@@ -45,18 +45,18 @@ public class MvcRazorHost : RazorEngineHost, IMvcRazorHost
         /// </summary>
         /// <param name="root">The path to the application base.</param>
         public MvcRazorHost(string root) :
```

Comment:
The WebRootFileSystemProvider is rooted at `~/wwwroot` versus Views are allowed to be found from `~/` or `~/Views`. Unfortunately this doesn't act as a drop in replacement.